templates/projects/project_page.html

```diff
@@ -2,66 +2,79 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 {% block main %}
-<div class="project-detail">
     {% include 'includes/breadcrumbs.html' with breadcrumbs=page.breadcrumbs current_page=page %}
     {% include 'includes/project_hero.html' %}
 
-    <div class="streamfields-wrapper">
-        {% if page.body %}
-            {% include_block page.body %}
-        {% endif %}
-    </div>
+    <div class="content-width grid-standard project-page">
+        <div class="project-page__main-content">
+            <div class="streamfields-wrapper">
+                {% if page.body %}
+                    {% include_block page.body %}
+                {% endif %}
+            </div>
+        </div>
+        <div class="project-page__side-content">
+            {% if page.related_links.all %}
+                <div class="project-page__side-content-block">
+                    <h2>Links</h2>
+                    <ul>
+                        {% for link in page.related_links.all %}
+                            {% with link_type=link.type.name %}
+                            <li class="{{ link_type|lower }}">
+                                <a href="{{ link.url }}">
+                                    {{ link_type }}
+                                </a>
+                            </li>
+                            {% endwith %}
+                        {% endfor %}
+                    </ul>
+                </div>
+            {% endif %}
+
+            {% if page.current_memberships %}
+                <div class="project-page__side-content-block">
+                    <h2>Team</h2>
+
+                    {% for membership in page.current_memberships %}
+                        <div class="project-page__side-content-item">
+                            <h3>{{ membership.role }}</h3>
+                            <div>
+                                {% include 'includes/project_membership.html' with member=membership.person %}
+                            </div>
+                        </div>
+                    {% endfor %}
 
-<div class="links">
-  <ul>
-    {% for link in page.related_links.all %}
-        {% with link_type=link.type.name %}
-        <li class="{{ link_type|lower }}"><a href="{{ link.url }}">
-            {# Special case: display project URL to make it visible & memorable  #}
-            {% if link_type == 'Website' %}<span>Project Website</span> <span class="url">{{ link.display_url }}</span>
-        {% else %}{{ link_type }}{% endif %}</a>
-        </li>
-        {% endwith %}
-    {% endfor %}
-    </ul>
-</div>
+                    {% if page.alums %}
+                        <div class="project-page__side-content-item">
+                            <h3>Project Alum{{ page.alums|pluralize }}</h3>
```

Comment:
so disappointed that wagtail doesn't do a special latin gender & pluralize function to spit out the correct form of alumnus/alumnae/alumni